AuthenticatorBridgeKit/SharedCryptographyService.swift

```diff
@@ -54,7 +69,26 @@ public class DefaultAuthenticatorCryptographyService: SharedCryptographyService
 
     // MARK: Methods
 
-    public func decryptAuthenticatorItems(
+    public func decryptAuthenticatorItemDatas(
+        _ items: [AuthenticatorBridgeItemData]
+    ) async throws -> [AuthenticatorBridgeItemDataView] {
+        let key = try await sharedKeychainRepository.getAuthenticatorKey()
+        let symmetricKey = SymmetricKey(data: key)
+
+        return items.compactMap { data in
+            guard let item = data.model else { return nil }
+
+            return AuthenticatorBridgeItemDataView(
+                favorite: item.favorite,
+                id: item.id,
+                name: (try? decrypt(item.name, withKey: symmetricKey)) ?? "",
+                totpKey: try? decrypt(item.totpKey, withKey: symmetricKey),
+                username: try? decrypt(item.username, withKey: symmetricKey)
+            )
+        }
+    }
```

Comment:
🤔 I think this function is not needed and also is duplicating decryption logic from the model. And the `decryptAuthenticatorItemModels` can be used instead with the proposed change in `sharedItemsPublisher`

Reply:
👍 Great suggestion. Fixed in latest.